libyul/backends/wasm/WasmDialect.cpp

```diff
@@ -46,12 +46,22 @@ WasmDialect::WasmDialect():
 	})
 		addFunction(name, 2, 1);
 
+	m_functions["i64.lt_u"_yulstring].returns.front() = "i32"_yulstring;
+	m_functions["i64.gt_u"_yulstring].returns.front() = "i32"_yulstring;
+	m_functions["i64.le_u"_yulstring].returns.front() = "i32"_yulstring;
+	m_functions["i64.ge_u"_yulstring].returns.front() = "i32"_yulstring;
+	m_functions["i64.eq"_yulstring].returns.front() = "i32"_yulstring;
+	m_functions["i64.ne"_yulstring].returns.front() = "i32"_yulstring;
+
 	addFunction("i64.eqz", 1, 1);
+	m_functions["i64.eqz"_yulstring].returns.front() = "i32"_yulstring;
 
 	addFunction("i64.store", 2, 0, false);
+	m_functions["i64.store"_yulstring].parameters.front() = "i32"_yulstring;
 	m_functions["i64.store"_yulstring].sideEffects.invalidatesStorage = false;
```

Comment:
Storage?!

Reply:
`i63.store` writes to memory, and this line registers that it does not modify storage.

Reply:
Sorry, mistook false for true, haha :)